src/main/java/com/synopsys/integration/alert/provider/blackduck/collector/builder/BomEditMessageBuilder.java

```diff
@@ -191,6 +194,8 @@ public BomEditMessageBuilder(PolicyCommonBuilder policyCommonBuilder) {
                                                         .applyAllComponentAttributes(commonAttributes)
                                                         .applyNotificationId(notificationId);
                     ComponentBuilderUtil.applyComponentInformation(builder, blackDuckResponseCache, componentData);
+                    BlackDuckIssueTrackerCallbackUtil.createCallbackInfo(getNotificationType(), versionBomComponent)
+                        .ifPresent(builder::applyComponentItemCallbackInfo);
```

Comment:
Looks like there's some duplication going on here that we may want to cut back on. (Same as line 151)

Reply:
Yeah, this is related to the tech-debt I was talking about. We definitely need to clean up these classes.